packages/css/src/components/skip-link/skip-link.scss

```
@@ -0,0 +1,30 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+.amsterdam-skip-link {
+  background-color: var(--amsterdam-skip-link-background-color);
+  color: var(--amsterdam-skip-link-color);
+  display: flex;
+  font-family: var(--amsterdam-skip-link-font-family);
+  font-size: var(--amsterdam-skip-link-font-size);
+  font-weight: var(--amsterdam-skip-link-font-weight);
+  justify-content: center;
+  line-height: var(--amsterdam-skip-link-line-height);
+  outline-offset: var(--amsterdam-skip-link-outline-offset);
+  padding-block: 0.5rem;
+  padding-inline: 1rem;
+  text-align: center;
+  text-decoration: none;
+  width: 100%;
```

Is there a good reason to use flexbox here? I’ve had some trouble with flexbox on text elements in the past. Using block display seems to allow shaving off (`justify-content` and `width`) as well.

```suggestion
  background-color: var(--amsterdam-skip-link-background-color);
  color: var(--amsterdam-skip-link-color);
  display: block;
  font-family: var(--amsterdam-skip-link-font-family);
  font-size: var(--amsterdam-skip-link-font-size);
  font-weight: var(--amsterdam-skip-link-font-weight);
  line-height: var(--amsterdam-skip-link-line-height);
  outline-offset: var(--amsterdam-skip-link-outline-offset);
  padding-block: 0.5rem;
  padding-inline: 1rem;
  text-align: center;
  text-decoration: none;
```

---

I think I originally centered the text with `display: flex` and `justify-content`, but `display: block` and `text-align: center` does the same job